tests/test_models_variational.py

```diff
@@ -102,13 +102,11 @@ def myloss(parameters, circuit, target):
 def test_vqe(backend, method, options, compile, filename):
     """Performs a VQE circuit minimization test."""
     if (method == "sgd" or compile) and (
-        (backend.name != "pytorch") or (backend.platform != "tensorflow")
+        backend.platform not in ["tensorflow", "pytorch"]
```

Comment:
This could become a property of the backend, e.g.:
``` python
@property
def support_differentiation(self,):
    return False
```
to avoid checking precisely for which backend this works, but again might be out of scope here.

Reply:
I think the best thing for Qibo will be to remove all these tests about variational models.